pkg/backends/calico/client.go

```diff
@@ -348,6 +355,24 @@ func (c *client) OnInSync(source string) {
 	}
 }
 
+func (c *client) updateExternalIPNets() {
+	v1Str, _ := model.KeyToDefaultPath(model.GlobalBGPConfigKey{Name: "svc_external_ips"})
```

Comment:
Hm, is this right? I would have expected us to need to parse the value of the object, not the key.

I think we need to pass in the update / kvpair into this function and extract the CIDRs from the value. As it is, I think this is always going to return 0 CIDRs (unless I'm misunderstanding something)

Reply:
I believe this is fixed now (https://github.com/projectcalico/confd/pull/266/commits/f3ad0fafde25d46607cad32c0774e50f92692d2f)